src/mono/wasm/runtime-test.js

```diff
@@ -91,138 +78,192 @@ if (typeof performance == 'undefined') {
 	}
 }
 
+// get arguments
+let testArguments = [];
 try {
-	if (typeof arguments == "undefined")
-		arguments = WScript.Arguments;
-	load = WScript.LoadScriptFile;
-	read = WScript.LoadBinaryFile;
-} catch (e) {
-}
+	if (is_node) {
+		testArguments = process.argv.slice (2);
+
+	}else if (is_browser) {
+		// We expect to be run by tests/runtime/run.js which passes in the arguments using http parameters
+		const url = new URL (decodeURI (window.location));
+		for (let param of url.searchParams) {
+			if (param [0] == "arg") {
+				testArguments.push (param [1]);
+			}
+		}
 
-try {
-	if (typeof arguments == "undefined") {
-		if (typeof scriptArgs !== "undefined")
-			arguments = scriptArgs;
+	}else if (typeof arguments === "undefined") {
+		if (typeof scriptArgs !== "undefined") {
+			testArguments = scriptArgs;
+		
+		}else if (typeof WScript  !== "undefined" && WScript.Arguments){
+			testArguments = WScript.Arguments;
+		}
+	} else{
+		testArguments = arguments;
 	}
 } catch (e) {
+	console.error(e)
```

Comment:
Or if you decide not to put console.error in our fake console, this and other references in our codebase all need to become console.log

Reply:
I think it is useful to have more than just `console.log`. So I just went through and replaced some of the `console.log` with `console.error` haha. Thanks for bringing it up.